content/en/docs/multicluster-management/unbind-cluster.md

```diff
@@ -1,10 +1,32 @@
 ---
 title: "Unbind a Cluster"
-keywords: 'kubernetes, kubesphere, multicluster, hybrid-cloud'
+keywords: 'Kubernetes, KubeSphere, multicluster, hybrid-cloud'
 description: 'Describe how to unbind a cluster from the KubeSphere multiple cluster management platform'
-
-
+linkTitle: "Unbind a Cluster"
 weight: 5500
 ---
 
-TBD
+This tutorial demonstrates how to unbind a cluster from the central control plane of KubeSphere.
+
+## Prerequisites
+
+- You have enabled multi-cluster management.
+- You need an account granted a role including the authorization of **Clusters Management**. For example, you can log in the console as `admin` directly or create a new role with the authorization and assign it to an account.
+
+## Unbind a Cluster
+
+1. Click **Platform** in the top left corner and select **Clusters Management**.
+
+2. On the **Clusters Management** page, click the cluster that you want to remove from the central control plane.
+
+   ![clusters-management](/images/docs/multicluster-management/unbind-a-cluster/clusters-management.jpg)
+
+3. Go to **Basic Info** under **Cluster Settings**, check **I confirm I want to unbind the cluster** and click **Unbind**.
```

Comment:
It is better to guide users how to delete a unhealthy cluster via command line since it is impossible to unbind the cluster through the console. I remember the command is like this.

```
kubectl delete <cluster name>
```

Reply:
Really? I tried the step to use the **Unbind** button and it actually works. A member cluster was successfully unbound. And it can still work well individually after it is unbound. Or do you mean it is impossible to unbind an unhealthy cluster so we need to delete it directly? If that is the case, do we have any standards to see how the cluster can be deemed as "unhealthy" so that it could not be unbound?

Reply:
BTW, there is already a guide talking about [how to delete a cluster](https://kubesphere.io/docs/installing-on-linux/uninstall-kubesphere-and-kubernetes/). Perhaps the command in this guide is what you are looking for?

Reply:
No. What I said is a different case. For example, import a cluster with a wrong credential. You will be unable to open the cluster detail page, so you have no chance to see the unbind button. Then you have to manually to delete the member cluster (not the kk delete cluster which is to delete the cluster physically, while here is to unbind the cluster) by using the kubectl (not kk) command

```
kubectl delete <cluster name>
```

Reply:
Noted. I will add the command to explain the case. Thanks.